src/_guides/language/language-tour.md

```diff
@@ -1586,16 +1581,33 @@ As you’ve already seen, you can assign values using the `=` operator.
 To assign only if the assigned-to variable is null,
 use the `??=` operator.
 
+<?code-excerpt "test/language_tour/operators_test.dart (assignment)"?>
 {% prettify dart %}
-a = value;   // Assign value to a
-b ??= value; // If b is null, assign value to b;
-             // otherwise, b stays the same
+a = value; // Assign value to a
+b ??= value; // Assign value to b if b is null;
+// otherwise, b stays the same
 {% endprettify %}
 
 {% comment %}
 <!-- embed a dartpad when we can hide code -->
 https://gist.github.com/9de887c4daf76d39e524
 https://dartpad.dartlang.org/9de887c4daf76d39e524
+
+<?code-excerpt "test/language_tour/operators_test.dart (assignment-gist-main-body)"?>
+{% prettify dart %}
+void assignValues(int a, int b, int value) {
+  print('Initially: a == $a, b == $b');
+  a = value; // Assign value to a
+  b ??= value; // Assign value to b if b is null;
+  // otherwise, b stays the same
+  print('After: a == $a, b == $b');
+}
+
+main() {
+  assignValues(0, 0, 1);
+  assignValues(null, null, 1);
+}
+{% endprettify %}
```

Comment:
This isn't a new code excerpt shown in the page (notice that it is inside a comment). Rather, it is just showing the code that is (should be) in the Gist.